packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactEditText.kt

```diff
@@ -610,6 +611,16 @@ public open class ReactEditText public constructor(context: Context) : AppCompat
         }
   }
 
+  public fun maybeUpdateTypeface() {
+    if (!typefaceDirty) {
+      return
+    }
+
+    typefaceDirty = false
+    updateTypefaceAndFlags()
+    forceUpdateTypefaceAndFlagsOnLayout = true
```

Comment:
If I am reading this right, this means that, for any case where we have had a transaction (props are changed, and `onAfterUpdateTransaction()` called), then if we have changed any flags contributing to typeface, we force recreating the typeface, before, and after layout?

This seems like it would impact performance, and may not be the right way to solve the issue happening here.

Could you help me understand the mechanism, for what changes here, between the invalidations? That way, we could find a more durable solution.